features/scripts/create_package.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
```

Comment:
I don't think this needs bash & could be a bit safer with `-e` specified (exit with an error if a command fails)

```suggestion
#!/usr/bin/env sh -ex
```